src/types.js

```diff
@@ -303,6 +303,7 @@ export type HvComponentProps = {|
 
 export type HvComponent<Props> = ComponentType<Props> & {
   localName: LocalName | string,
+  localNameAliases: Array<LocalName | string>,
```

Comment:
When can this alias include a string other than the ones in `LocalName`?

Reply:
When the component is not part of the core (i.e. our own custom components)